gateway/src/templates/vvs.yml

```diff
@@ -0,0 +1,16 @@
+# how much the execution price is allowed to move unfavorably from the trade
+# execution price. It uses a rational number for precision.
+allowedSlippage: '1/100'
+
+# the maximum gas used to estimate gasCost for a pangolin trade.
+gasLimitEstimate: 150688
+
+# how long a trade is valid in seconds. After time passes Mad Meerkat will not
+# perform the trade, but the gas will still be spent.
+ttl: 300
+
+tradingTypes: ['EVM_AMM']
+
+contractAddresses:
+  mainnet:
+    routerAddress: '0x145677FC4d9b8F19B5D56d1820c48e0443049a30'
```

Comment:
```suggestion
    routerAddress: '0x145863Eb42Cf62847A6Ca784e6416C1682b1b2Ae'
```
This isn't correct.
![Screenshot 2022-10-12 135844](https://user-images.githubusercontent.com/31972210/197353638-eb6f14c7-cf69-4cc2-931a-4f166bee2e8a.png)